sympy/calculus/tests/test_singularities.py

```diff
@@ -11,12 +11,17 @@
 a = Symbol('a', negative=True)
 b = Symbol('b', positive=True)
 
-def test_singularities():
-    x = Symbol('x', real=True)
-
-    assert singularities(x**2, x) == ()
-    assert singularities(x/(x**2 + 3*x + 2), x) == (-2, -1)
 
+def test_singularities():
+    x = Symbol('x')
+    y = Symbol('y', complex=True)
```

Comment:
I don't think there is any need to explicitly define the symbol `y` with assumption `complex=True`.